crates/sui-benchmark/Cargo.toml

```diff
@@ -29,6 +29,7 @@ multiaddr = "0.14.0"
 crossterm = "0.23.2"
 rand = "0.8.5"
 base64 = "0.13.0"
```

Comment:
base64ct: please pin the most recent version 1.5.2

Unrelated to this PR, but we should avoid using both base64 libs (atm we prefer base64ct for the malleability guarantees)

Reply:
I agree with you, but I do think it is totally fine to keep it in this crate though, as it is just used for benchmarks.

Reply:
1. I didn't add new tests, as there are already extensive tests covering the behavior of the Committee struct.
2. I'll run some for ed25519, but honestly, the benchmarks fluctuate a lot so I'm unsure whether we'll see anything.
3. I gave you the wrong answer last time - I checked out the ed25519 code again and they store both the compressed and uncompressed points, so there is no difference in the implementation of ed25519 and bls12381.